charts/rekor/values.yaml

```diff
@@ -12,7 +12,8 @@ redis:
     - "yes"
   name: redis
   image:
-    repository: docker.io/redis
+    registry: docker.io
+    repository: redis
     pullPolicy: IfNotPresent
     version: "5.0.10"
```

Comment:
We should use the digest here.`sha256: 4f226f7d68a9880b3f4952d53d5d57c0f3b0c4941b455f7279a12c63c9004a00`

Reply:
@hectorj2f Updated to include the value of the digest. `sha256:0a0d563fd6fe5361316dd53f7f0a244656675054302567230e85eb114f683db4` was the value used as its the amd64 digest instead of the arm version you referenced (why dockerhub defaults to arm is another story :) ). 

Thanks again for the suggestion to move over to digests for this image!